modyn/supervisor/internal/triggers/embedding_encoder_utils/embedding_encoder_downloader.py

```diff
@@ -24,12 +25,13 @@ class EmbeddingEncoderDownloader:
 
     def __init__(
         self,
-        modyn_config: dict,
+        modyn_config: ModynConfig,
         pipeline_id: int,
         base_dir: pathlib.Path,
         model_storage_address: str,
     ):
-        self.modyn_config = modyn_config
+        # TODO(MaxiBoether): Update this class to use the model
```

Comment:
Not sure how much effort this is here, I remember that I have tried to integrate the models into the database components when introducing the pydantic models, that was a nightmare with all the tests that don't set all required fields. I wouldn't mind if we remove this TODO.